### PR TITLE
rm: add Korean translation

### PR DESCRIPTION
**Pull request by sooohka**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ko/common/rm.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# rm
+
+> 파일 혹은 폴더를 삭제.
+> 더 많은 정보: <https://www.gnu.org/software/coreutils/rm>.
+
+- 임의의 경로에서 파일을 제거:
+
+`rm {{path/to/file}} {{path/to/another/file}}`
```
Comment (marchersimon) [inline pages.ko/common/rm.md:8]: Do you think it would be helpful if you'd translate arguments too?

Comment (blueskyson) [inline pages.ko/common/rm.md:8]: Thanks for contribution. Please translate `path/to/something` into Korean as well.

File: pages/common/rscript.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Rscript
```
Comment (marchersimon) [inline pages/common/rscript.md:1]: Why is this page being added, even though it was added in another PR by another author? (#6686)

Comment (sooohka) [reply pages/common/rscript.md]: not sure why this file is in 😞  i'll delete it

File: pages.ko/common/rm.md (hunk 1 above)
Comment (sooohka) [reply pages.ko/common/rm.md]: resolved!

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @sooohka

**Pull request merged**